core/src/dird/dbcheck.cc

```diff
@@ -715,7 +716,7 @@ static void repair_bad_paths()
       db->EscapeString(nullptr, esc_name, name, len);
       Bsnprintf(buf, sizeof(buf), "UPDATE Path SET Path='%s' WHERE PathId=%s",
                 esc_name, edit_int64(id_list.Id[i], ed1));
-      if (verbose > 1) { printf("%s\n", buf); }
+      if (g_verbose > 1) { printf("%s\n", buf); }
```

Comment:
I think verbose can currently only be set to 0 or 1, right ?
We should probably create a task that allows us to specify verbosity levels somehow, assuming that it was previously possible to do so.

Reply:
It has been like this since we forked. So we can surely fix this, but I'd just make `g_verbose` a bool and remove all the code that is guarded with `g_verbose > 1`.